spec/models/publication_spec.rb

```diff
@@ -152,7 +152,6 @@
     it 'should sync existing authors in the pub hash to contributions in the db' do
       publication.pub_hash = { authorship: [{ status: 'new', sul_author_id: author.id }] }
       publication.send(:update_any_new_contribution_info_in_pub_hash_to_db)
-      publication.save
       expect(publication.contributions.size).to eq(1)
```

Comment:
Why removing the `publication.save` in several places?

Reply:
It isn't part of what is being tested here.  I.E., `publication.contributions.size` and `publication.contributions.last` don't rely on having been saved (and shouldn't!)

Reply:
I know this is actually in the pub spec, so it's sorta OK, but I can't help wondering if the `send` could be removed and replaced with the `save` instead.  Would be cool if the spec respected the callbacks too.  In this case, seems like the spec is just asking pub to save a contribution and it should be doing that in a before block or something, unless the spec is really targeting that private method specifically.   (Can't see all the context from this github view.)  If it's generally testing the callbacks as a whole, it should use `save`, but if it's really focused on just that private method, OK, use the send.  If it's the latter, seems like it should be using an rspec change pattern, like
```
expect { pub-with-authorship-to-save-a-contribution }.to change { Contribution.count }
```

Reply:
No, the `send` IS the method being tested.  This is a unit test on the private function that is part of the save callbacks.  `.save` is tested elsewhere.